### PR TITLE
Update Pace to the latest version, and remove duplicate files in app/assets.

### PR DESCRIPTION
**Pull request by caseyscarborough**
Update Pace to version 0.5.6, and removed the duplicate files in `app/assets` in favor of `vendor/assets`. Added minor stylistic updates to `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,51 +9,48 @@ Automatic web page progress bar.
...
+```
 
 or you can install from latest build:
 
-    gem 'pace-rails', git: 'git@github.com:yovu/pace-rails.git'
+```ruby
+gem 'pace-rails', git: 'git@github.com:yovu/pace-rails.git'
+```
 
 ## Usage
```
Comment (hughes123b) [inline README.md:18]: []()

File: app/assets/stylesheets/pace/pace-theme-big-counter.css [hunk 2]
```diff
@@ -1,35 +0,0 @@
-/* This is a compiled file, you should be editing the file in the templates directory */
-.pace {
-  -webkit-pointer-events: none;
-  pointer-events: none;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  user-select: none;
-}
-
-.pace.pace-inactive .pace-progress {
```
Comment (hughes123b) [inline app/assets/stylesheets/pace/pace-theme-big-counter.css:10]: People I need $$

Comment (hughes123b) [reply app/assets/stylesheets/pace/pace-theme-big-counter.css:10]: Poopy

File: vendor/assets/javascripts/pace/pace.js [hunk 3]
```diff
@@ -426,15 +426,19 @@
...
           monitorXHR(req);
           return req;
         };
-        extendNative(window.XDomainRequest, _XDomainRequest);
+        try {
+          extendNative(window.XDomainRequest, _XDomainRequest);
+        } catch (_error) {}
       }
       if ((_WebSocket != null) && options.ajax.trackWebSockets) {
         window.WebSocket = function(url, protocols) {
```
Comment (hughes123b) [inline vendor/assets/javascripts/pace/pace.js:440]: Yea

**Comment by acidtib** [comment]
@hughes123b ?

**Pull request merged**